core/src/main/java/hudson/model/Queue.java

```diff
@@ -2461,8 +2461,8 @@ public int compareTo(WaitingItem that) {
         }
 
         public CauseOfBlockage getCauseOfBlockage() {
-            long diff = timestamp.getTimeInMillis() - System.currentTimeMillis();
-            if (diff > 0)
+            long diff = System.currentTimeMillis() - timestamp.getTimeInMillis();
+            if (diff <= 10000)
                 return CauseOfBlockage.fromMessage(Messages._Queue_InQuietPeriod(Util.getTimeSpanString(diff)));
             else
                 return CauseOfBlockage.fromMessage(Messages._Queue_Unknown());
```

Comment:
This message probably just needs massaging. The reason appears to be a late queue maintenance moving the `WaitingItem` into its next state. So something like "Finished waiting" would be better for low thresholds.